modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -0,0 +1,260 @@
+[[content-lifecycle]]
+= Content Lifecycle Management Examples
+
+This section contains some common examples of how you can use content lifecycle management.
+Use these examples to build your own personalised implementation.
+
+
+
+[#create-proj-for-patch-cycle]
+== Creating a Project for a Monthly Patch Cycle
+
+
```

Comment:
```suggestion
A project for a monthly patch cycle consists of:
```
Or something similar. We need an intro sentence, IMO.  (And we probably do not want 3 empty lines before the list starts.)